styles.css

```diff
@@ -0,0 +1,391 @@
+*{
+	box-sizing: border-box;
+}
+
+body {
+    max-width: 80rem;
+    margin: auto;
+}
+
+
+header {
+	height:10%;
+	background-color:#093C60;
+	color:white;
+    padding: 5% 13%;
+    max-height: 400px;
+    position: relative;
+}
+
+header h1 {
+	font-size: 1.1em;
+	font-family: 'Courgette', cursive;
+	margin-top: -10px;
```

Comment:
I'm leery about all these negative margins. Maybe dial down the padding on the header element? I'd rather have positive margin on the image. Negative margin can be tricky